ui/app/hooks/useTransactionDisplayData.js

```diff
@@ -143,7 +143,7 @@ export function useTransactionDisplayData (transactionGroup) {
     primarySuffix = isViewingReceivedTokenFromSwap
       ? currentAsset.symbol
       : initialTransaction.sourceTokenSymbol
-    primaryDisplayValue = swapTokenValue
+    primaryDisplayValue = removeDash(swapTokenValue)
```

Comment:
I think this would give us the wrong answer in any case where the dash exists. It is important to know whether this is a negative value or not.

For example, in this case I encountered:

![plus-minus](https://user-images.githubusercontent.com/2459287/97370823-d99a6c00-1892-11eb-827c-3b4449723bcf.png)

In this case, `isViewingReceivedTokenFromSwap` was `true`, so the prefix would be `+`. But we're _losing_ ETH here, so the correct prefix is `-`.

I think we want to do two things here:

* "reverse" the prefix used (e.g. add or remove a dash, or multiply by `-1`, whichever) if `isViewingReceivedTokenFromSwap` is false, and
* add `+` if the resulting number is positive.

Reply:
> But we're _losing_ ETH here, so the correct prefix is `-`.

Wait if you swap a token to ETH, we are losing ETH?

Reply:
>Wait if you swap a token to ETH, we are losing ETH?

Potentially, yep! In this case, I lost money because the transaction failed on-chain. I had to pay the gas fee but I got nothing in return.